ticketbuyer/buyer.go

```diff
@@ -559,7 +589,7 @@ func (t *TicketPurchaser) Purchase(height int64) (*PurchaseStats, error) {
 	// have enough moneys.
 	notEnough := func(bal dcrutil.Amount, toBuy int, sd dcrutil.Amount) bool {
 		return (bal.ToCoin() - float64(toBuy)*sd.ToCoin()) <
-			t.cfg.BalanceToMaintain
+			float64(balanceToMaintainAmt)
```

Comment:
should be balanceToMaintainAmt.ToCoin() instead.

Got this on testnet: 
`Not buying because spendable balance would be 34849.05986393but balance to maintain is 12926.13689973 DCR`